package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "webdb-challenge",
+  "version": "1.0.0",
+  "description": "This challenge allows you to practice the concepts and techniques learned over the past Sprint and apply them in a concrete project.",
+  "main": "index.js",
+  "scripts": {
+    "server": "nodemon index.js",
+    "start": "java index.js"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/BrockFalfas/webdb-challenge.git"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/BrockFalfas/webdb-challenge/issues"
+  },
+  "homepage": "https://github.com/BrockFalfas/webdb-challenge#readme",
+  "dependencies": {
+    "dotenv": "^8.0.0",
+    "express": "^4.17.0",
+    "helmet": "^3.18.0",
+    "knex": "^0.16.5",
+    "morgan": "^1.9.1",
+    "sqlite3": "^4.0.8"
+  },
+  "devDependencies": {
+    "knex-cleaner": "^1.1.4",
```

Comment:
OK so you did install knex-cleaner, you just didn't use it, I know seeding was a stretch though.